ms-docs/JBoss EAP on Azure.MD

```diff
@@ -1,170 +1,165 @@
 
-* Hibernate Applications - Developers and administrators can develop and deploy JPA/Hibernate applications with Red Hat JBoss Enterprise Application Platform. Hibernate Core is an object-relational mapping framework for the Java language. It provides a framework for mapping an object-oriented domain model to a relational database, allowing applications to avoid direct interaction with the database. Hibernate EntityManager implements the programming interfaces and lifecycle rules as defined by the [Java Persistence 2.1 specification](https://www.jcp.org/en/jsr/detail?id=338). Together with Hibernate Annotations, this wrapper implements a complete (and standalone) JPA persistence solution on top of the mature Hibernate Core. Additional information on [Developing Hiberate Applications](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/developing_web_services_applications/index) is available from Red Hat. 
+* Hibernate Applications - Developers and administrators can develop and deploy JPA/Hibernate applications with JBoss EAP. Hibernate Core is an object-relational mapping framework for the Java language. It provides a framework for mapping an object-oriented domain model to a relational database, allowing applications to avoid direct interaction with the database. Hibernate EntityManager implements the programming interfaces and lifecycle rules as defined by the [Java Persistence API (JPA) 2.1 specification](https://www.jcp.org/jsr/detail?id=338). Together with Hibernate Annotations, this wrapper implements a complete (and standalone) JPA solution on top of the mature Hibernate Core. Additional information on [Developing Hibernate Applications](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/development_guide/java_persistence_api) is available from Red Hat. 
 
 ### Migrate from on-premises to Azure
-Azure Marketplace offer of JBoss EAP on RHEL will install and provision on Azure VMs or VMSS in less than 20 minutes. You will have a Load Balancer (LB), Private IP for LB and VMs, Virtual Network (VNet) with a single subnet, cluster or stand-alone setup, Network Security Group (NSG), and a sample Java app. 
+Azure Marketplace offer of JBoss EAP on RHEL will install and provision Azure VMs or VMSS in less than 20 minutes. You will have a Load Balancer (LB), Private IP for LB and VMs, Virtual Network (VNet) with a single subnet, cluster or stand-alone setup, Network Security Group (NSG), and a sample Java app. 

```

Comment:
The Azure marketplace image will provide only JBoss EAP on RHEL VM. The other configurations mentioned here are about the templates in GitHub which contain JBoss EAP on RHEL VM (Standalone, Clustered) and RHEL VMSS. So please suggest the required changes here.